instrumentation/p6spy/src/test/java/brave/p6spy/TracingJdbcEventListenerTest.java

```diff
@@ -31,6 +31,7 @@
 
   @Mock Span span;
   String url = "jdbc:mysql://127.0.0.1:5555/mydatabase";
+  String urlWithWhiteSpace = url + ";parameter=with white space;foo=bar";
```

Comment:
Can you use a realistic example of a parameter with whitespace?

Reply:
Sure !

Reply:
Done !